### DISCUSSION TITLE
Registration emails not sending and logs not helping

### DISCUSSION THREAD

**Discussion by AwesomeStickz** [WeblateOrg/weblate#10402] [Q&A]
Hi, I've been having issues sending registration emails, I tried `sendtestemail` and that works, and also contacting through /contact sends me an email from the same email ID as well but when I try to register, I'm not getting an email, I looked at the docker compose logs as well and nothing seems weird, is there any suggestions for me to debug this?

**Comment by AwesomeStickz** [comment]
Well I tried restarting using `docker-compose restart`, a few mins later those emails were sent by themselves, so I think it _might_ be some celery queues issue? But I have a question, does anyone have any idea why this happened in this first place and how I could prevent this from happening again?

**Reply by nijel** [reply]
Most likely one the Celery processes was not running in the container. Maybe it was killed by OOM?

**Reply by AwesomeStickz** [reply]
Hm, I don't think my server has ever went to very low available memory state but I often had no (storage) space left so I think that could be one of the reasons 🤔
